pkgs/by-name/th/thud/package.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, installShellFiles
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "thud";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "donovanglover";
+    repo = "thud";
+    rev = version;
+    hash = "sha256-3MxmVKs0huXPnL9mqDniaIarkAvJmwSOMii2ntXtOos=";
+  };
+
+  cargoHash = "sha256-Hk3HlcA253FAA9hw5p9W+Mvec84zLo7bEmM2/BbmjiM=";
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  postInstall = ''
+    mkdir -p $out/share/thumbnailers
+    substituteAll ${./thud.thumbnailer} $out/share/thumbnailers/thud.thumbnailer
```

I don't think we need to vendor `thud.thumbnailer`.

How about

```suggestion
    mkdir -p $out/share/thumbnailers
    substitute assets/thud.thumbnailer $out/share/thumbnailers/thud.thumbnailer \
      --replace "thud" "$out/bin/thud"
```

or

```suggestion
    install -Dm644 assets/thud.thumbnailer $out/share/thumbnailers/thud.thumbnailer
    substituteInPlace $out/share/thumbnailers/thud.thumbnailer --replace "thud" "$out/bin/thud"
```

---

Fixed